File: src/main/java/Main/ItemClasses/Miscellaneous.java
package Main.ItemClasses;

public class Miscellaneous extends Item{
    public Miscellaneous(String name, String description, int weight, int valueInGold, int amount) {
        super(name, description, weight, valueInGold, amount);
        this.standardIconName = "Log of Wood.png";
    }

    public Miscellaneous() {
        super();
        this.standardIconName = "Log of Wood.png";
    }
}

File: src/main/kotlin/itemClasses/Miscellaneous.kt
package itemClasses

import kotlinx.serialization.Serializable

@Serializable
class Miscellaneous() : Item() {
    init {
        standardIconName = "Log of Wood.png"
    }

    constructor(name: String, description: String, weight: Int, valueInGold: Int, amount: Int) : this() {
        this.name = name
        this.description = description
        this.weight = weight
        this.valueInGold = valueInGold
        this.amount = amount
    }
}
